packages/nocodb/Dockerfile.local

```diff
@@ -49,7 +49,7 @@ RUN apk --update --no-cache add \
     curl \
     jq
 
-# Copy packaged production code & main entry file
+# Copy production code & main entry file
 COPY --from=builder /usr/src/appEntry/ /usr/src/appEntry/
```

Comment:
We have to copy build and `node_modules` to `/usr/src/app/`

`L#31` can be removed and here we can add

```
COPY --from=builder /usr/src/app/ /usr/src/app/
```

Reply:
Ah sorry, I mixed up `/usr/src/app/` and `/usr/src/appEntry/` 🙈. Should be fine now.